scrapi/harvesters/ut_chattanooga.py

```diff
@@ -0,0 +1,29 @@
+'''
+Harvester for the UTC Scholar for the SHARE project
+
+Example API call: http://scholar.utc.edu/do/oai/?verb=ListRecords&metadataPrefix=oai_dc
+'''
+from __future__ import unicode_literals
+
+from scrapi.base import OAIHarvester
+
+
+class Ut_chattanoogaHarvester(OAIHarvester):
+    short_name = 'ut_chattanooga'
+    long_name = 'University of Tennessee at Chattanooga'
+    url = 'http://scholar.utc.edu'
+
+    base_url = 'http://scholar.utc.edu/do/oai/'
+    property_list = ['type', 'format', 'identifier', 'date', 'source', 'setSpec', 'publisher', 'language', 'rights']
+
+    approved_sets = [
+        u'honors-theses',
+        u'theses',
+        u'student-research',
+        u'ebm-symposium',
+        u'conferences',
+        u'journals',
+        u'jafh'
+    ]
```

Comment:
Hi @kms6bn ! Yep, I'd go ahead and remove the ones you added.  We usually let the providers let us know what they'd like in SHARE as they know best what they have (as some would prefer undergrad or other content not make it).